Formula/dotnet.rb

```diff
@@ -40,14 +35,9 @@ class Dotnet < Formula
   def install
     ENV.append_path "LD_LIBRARY_PATH", Formula["icu4c"].opt_lib if OS.linux?
 
-    # Arguments needed to not artificially time-limit downloads from Azure.
-    # See the following GitHub issue comment for details:
-    # https://github.com/dotnet/source-build/issues/1596#issuecomment-670995776
-    system "./build.sh", "/p:DownloadSourceBuildReferencePackagesTimeoutSeconds=N/A",
-                         "/p:DownloadSourceBuiltArtifactsTimeoutSeconds=N/A"
-
+    system "./build.sh", "--configuration Release"
```

Comment:
Looks like this installs pre-built binaries, no? That probably explains all the Linux failures. We probably want to follow these steps instead:

https://github.com/dotnet/installer#building-source-build

Reply:
You're quite right. I'll fix.

Reply:
As per https://github.com/dotnet/installer/issues/12871#issuecomment-1011523265, I'm currently stuck. I'll progress this PR once the build is successful.

Reply:
Thanks for working on this. I commented on that issue, but I'm not sure if that's what you're still stuck on.

Reply:
I added changes that I needed for successful ARM64 Monterey build.
Still need some similar fixes to get Intel build working.

Don't know about Linux as I can't build locally in container. In theory, should be more stable as upstream runs on Linux and Arch/Fedora/etc have already updated.